actionview/lib/action_view/renderer/partial_renderer.rb

```diff
@@ -307,7 +307,8 @@ def render(context, options, block)
       if @collection
         render_collection
       else
-        instrument(:partial) do
+        instrument(:partial) do |payload|
+          @view.instance_variable_set(:@log_payload_for_partial_render, payload)
```

Comment:
Thinking about this some more, if we are to provide fine grained `[cache hit]` misses per partial render will this do?

In other words what's the life span of both `@view` and `@log_payload_for_partial_render`? If one variable sticks around after the other, then we will generate wrong information. Consider:

```ruby
# app/views/customers/show.html.erb
render :customer # => [cache hit]
render :todo_list  # => Does this render inherit the [cache hit] from the previous render?
```